### DISCUSSION TITLE
Shift Mouse-wheel for faster scroll ?

### DISCUSSION THREAD

**Discussion by meteorquake** [sumatrapdfreader/sumatrapdf#3336] [Ideas]
I use the mouse-wheel for scrolling on a document page, and doing it over the scroll-bar seems to give a faster scroll when needed.
* It would be extremely good to be able to shift-mouse-wheel in the main page to get a faster scroll, without having to move the mouse to the side.
Cheers, David

**Comment by GitHubRulesOK** [comment]
Use the Accelerator key (<kbd>Alt</kbd> key)

From Manual 
>Alt + Scroll Wheel - Increase vertical scroll wheel steps (faster)

**Comment by meteorquake** [comment]
That was simple - I was on the right track! many thanks! David
